### ISSUE TITLE
WHITE BALANCE

### ISSUE THREAD

**Issue by studiovideo54** [bitfocus/companion-module-panasonic-ptz#14]
Hello, why preset white balance is not present?
Many thanks

Dory

**Comment by McHauge** [comment]
@studiovideo54 Cause they haven't been added yet... it's possible to add them in the new development branch, but in the current main branch would be hard, as there are 3-5 types of ways Panasonic does white balance on its PTZ's.

**Comment by jbilyj** [comment]
It would be great to see at least variable colour temperature control added. I know this would extremely useful for the AW-HE130's that we run.

**Comment by McHauge** [comment]
Yep, it's definitely on the list, just haven't had any camera i could take home to test on for a while

**Comment by cpuguru07** [comment]
Also needing White Balance/color controls.

**Comment by studiovideo54** [comment]
Yes for my panasonic awhe130

 

Doriano Costantino

Studiovideo54

Cell. +39 338.8747472

***@***.***

 

 

 

From: cpuguru07 ***@***.***>
Reply to: bitfocus/companion-module-panasonic-ptz ***@***.***>
Date: Wednesday, 19 October 2022 at 23:18
To: bitfocus/companion-module-panasonic-ptz ***@***.***>
Cc: studiovideo54 ***@***.***>, Mention ***@***.***>
Subject: Re: [bitfocus/companion-module-panasonic-ptz] WHITE BALANCE (#14)

 

Also needing White Balance/color controls.

**Comment by premultiply** [comment]
This module is deprecated and is no longer actively maintained.

Implemented in its successor, https://github.com/bitfocus/companion-module-panasonic-cameras.
